lua/wikis/commons/Class.lua

```diff
@@ -110,40 +104,14 @@ function Class._wrapFunction(f, options)
 			)
 
 		if shouldRewriteArgs then
-			local namedArgs, indexedArgs = Class._frameToArgs(frame, options)
-			if namedArgs then
-				return f(namedArgs, unpack(indexedArgs))
-			else
-				return f(unpack(indexedArgs))
-			end
+			local args = Arguments.getArgs(frame, options)
+			return f(args)
 		else
 			return f(...)
```

Comment:
We shouldn't use the spread operator here if we want to keep the "promise" that functions can be called the same from Lua and wikicode, right?

Reply:
as of now we still need it
there exist exported functions that are only supposed to be accessed by lua (e.g. expecting `string[], table`) which were never accessible from invoke to begin with ...
if we removed the `...` those would break

longterm we likely should refactor those cases
but it is lots of work
(alone the cleanup i am doing in #5824 is already a ton of work, currently don't have the time to do this extra step too)

Reply:
OOS, added as todo in #5818

Reply:
first step for resolving this without breaking stuff via #5828 
will resolve the comment here as it is oos for this PR